test/client/base_view.test.js

```diff
@@ -133,7 +132,7 @@ describe('Base/View', function () {
 
   describe('attachOrRender', function () {
     beforeEach(function () {
-      this.$el = $('<div/>');
+      this.$el = global.window.$('<div/>');
```

Comment:
@mdimas @jmerrifield - this is the source of the wrong document error. stupid document.